### PR TITLE
theme updation for drag and drop in file input

### PR DESCRIPTION
**Pull request by rajatofficial**

### CODE REVIEW

File: .github/workflows/publish.yml [hunk 1]
```diff
@@ -47,10 +47,18 @@ jobs:
       run: |
         npm version patch -m "Bump version to %s"
 
+    - name: Authenticate with NPM Registry
```
Comment (amit-mnnit) [inline .github/workflows/publish.yml:50]: any reason to change the workflow file?

File: src/components/fileinput/_fileinput.scss [hunk 2]
```diff
@@ -110,4 +117,36 @@
...
+.cmp-adaptiveform-fileinput__dragtext {
+  padding-bottom: 10px;
+  padding-bottom: 20px;
+}
+
+.cmp-adaptiveform-fileinput__icon {
+  height: 50px;
+  width: 50px;
+  background-size: cover;
+  background-image: url('');
```
Comment (amit-mnnit) [inline src/components/fileinput/_fileinput.scss:137]: ?

File: src/components/fileinput/_fileinput.scss [hunk 3]
```diff
@@ -110,4 +117,36 @@
...
+}
+
+.cmp-adaptiveform-fileinput__filesize {
+  float: left;
```
Comment (amit-mnnit) [inline src/components/fileinput/_fileinput.scss:141]: not working for RTL

File: src/components/fileinput/_fileinput.scss [hunk 4]
```diff
@@ -110,4 +117,36 @@
...
+  text-shadow: 0 1px #fff;
+  opacity: .2;
+  font-size: 1.25rem;
+  font-weight: 700;
+  line-height: 1;
+  margin-right: 5px;
+}
+
+.cmp-adaptiveform-fileinput__fileendcontainer {
+  float: right;
```
Comment (amit-mnnit) [inline src/components/fileinput/_fileinput.scss:151]: not working for RTL

**Review by amit-mnnit — CHANGES_REQUESTED** [review]
Please check comments

File: src/components/fileinput/_fileinput.scss (hunk 3 above)
Comment (rajatofficial) [reply src/components/fileinput/_fileinput.scss]: removed will be handled automatically by the parent container

File: src/components/fileinput/_fileinput.scss (hunk 4 above)
Comment (rajatofficial) [reply src/components/fileinput/_fileinput.scss]: removed will be handled automatically by the parent container

File: .github/workflows/publish.yml (hunk 1 above)
Comment (rajatofficial) [reply .github/workflows/publish.yml]: removed!

**Review by amit-mnnit — APPROVED** [review]

**Pull request merged**